tests/Microsoft.Bot.Connector.Tests/Authentication/JwtTokenExtractorTests.cs

```diff
@@ -63,14 +63,16 @@ public async Task Connector_TokenExtractor_RequiredEndorsementsPresent_ShouldVal
             Assert.True(claimsIdentity.IsAuthenticated);
         }
 
-        //[Fact]
-        //public async Task Connector_TokenExtractor_RequiredEndorsementsPartiallyPresent_ShouldNotValidate()
-        //{
-        //    var configRetriever = new TestConfigurationRetriever();
-
-        //    configRetriever.EndorsementTable.Add(KeyId, new HashSet<string>() { RandomEndorsement, ComplianceEndorsement, TestChannelName });
-        //    await Assert.ThrowsAsync<UnauthorizedAccessException>(async () => await RunTestCase(configRetriever, new string[] { ComplianceEndorsement, "notSatisfiedEndorsement" }));
-        //}
+        /*
```

Comment:
As this is a commented Test, shouldn't we remove it???

Reply:
Yes, I guess we should. I'll remove it.